### ISSUE TITLE
Question about the results presented in the paper 

### ISSUE THREAD

**Issue by waychin-weiqin** [oripress/CCC#10]
Hi 👋🏼 @oripress ,

Thank you for the interesting work and for releasing the benchmark. I have a question about the results presented in the paper, specifically those shown in Figures 1(b), 3, and 4.

If I understand correctly, RDumb resets the entire model to its initial pretrained state at every T (T=1000) interval. If this is accurate, why doesn't RDumb's performance drop to the level of the pretrained model at each reset interval? In these figures, RDumb consistently achieves higher accuracy than the pretrained model.

Additionally, I am curious about which parameters are updated when ViT is used as the backbone, given that ViT models do not have batch norm layers.

Thank you for your time in answering these questions.

**Comment by oripress** [comment]
Hi, 
The plots show accuracy over a running window of 781 batches (around 50k) images, which is why RDumb remains consistently higher than the pretrained model. 

With ViT, we update the LayerNorm params.

Hope this helps!

**Comment by waychin-weiqin** [comment]
Hi again, 

Is the implementation for computing the accuracies with the running window different from the one included in the eval.py (line37-58)? If so, could you kindly guide me on how to compute the accuracies in order to generate plots like those in Figures 1(b), 3, and 4? 

Thank you in advance for your help!

**Comment by oripress** [comment]
Sure, in the eval code you get an accuracy per batch. 

Then you have a list of accuracies = [batch_1_acc, batch_2_acc, ..., batch_n_acc]
Now we can use this function to calculate sliding windows

```python
def sliding_window_averages(lst, window_size):
    averages = [sum(lst[i:i + window_size]) / window_size for i in range(len(lst) - window_size + 1)]
    return averages[::window_size]
    
averages = sliding_window_averages(accuracies, 781)


Originally I did it with pandas dataframe functions, but this should be equivalent.
